apps/judicial-system/web/src/utils/hooks/useS3Upload/useS3Upload.ts

```diff
@@ -14,12 +17,20 @@ import {
   DeleteFileMutationDocument,
   DeleteFileMutationMutation,
   DeleteFileMutationMutationVariables,
+  LimitedAccessCreateFileMutationDocument,
+  LimitedAccessCreateFileMutationMutation,
```

Comment:
Is there a way to write our mutations so that the graphql schema generates them without naming them MutationMutation? So just LimitedAccessCreateFileMutation

Reply:
Yes there is and I thought about making that change in this PR but then decided the PR is big enough as is. I suggest we take this as a cool down task and I will add one to our inbox.